### ISSUE TITLE
Trouble building

### ISSUE THREAD

**Issue by samuelcole** [andyearnshaw/Intl.js#95]
```
Running "bundle_jsnext:dest" (bundle_jsnext) task
INFO: Using a global wrapper module for "src/main.js", exporting namespace "IntlPolyfill"
Fatal error: Error writing bundle in "dist/Intl.js": Error: missing module import from ./core for path: ./cldr
```

I ran all the `cldr` tasks and everything. What am I missing?

**Comment by kate2753** [comment]
@samuelcole, @caridy missed to check in `src\cldr.js` file during refactor in #85 . I've left a [comment](https://github.com/andyearnshaw/Intl.js/pull/85#issuecomment-96828774) on that pull request, but never heard back.

**Comment by samuelcole** [comment]
so we're out of luck until he pushes that? that's a bummer...

this code looks great though!

**Comment by kate2753** [comment]
You could probably re-create what original `cldr.js` file was based on [dist\Intl.js](https://raw.githubusercontent.com/andyearnshaw/Intl.js/master/dist/Intl.js). Anything prefixed with `$$cldr$$` was originally in `src\cldr.js` file. That's pretty cumbersome though. Having @caridy check in the original file is much better way to this this! :)

**Comment by andyearnshaw** [comment]
I think @caridy has been away and I'm not sure if that is still the case.  Maybe @ericf would know more.

**Comment by jasonmit** [comment]
He is still away @andyearnshaw

**Comment by samuelcole** [comment]
still missing?

**Comment by andyearnshaw** [comment]
Fixed by #101.
